pyspades/constants.py

```diff
@@ -72,9 +73,11 @@
 HIT_TOLERANCE = 5.0
 CLIP_TOLERANCE = 10
 
+SPADE_TOOL_3X = 4
```

Comment:
I like the separation of rapid detection by tool. But I'm not a huge fan of having this value here. It's bound to be confusing because it's separate from the other tool values, only used by speedhack detection and invalid in the rest of the code.

I think there's two ways of possibly doing it better, depending on how much it interferes with the rest of the code: having a separate `enum.enum` for events that are rate limited, or getting rid of the dict and just having straightforward constants for the rate limits.